README.md

```diff
@@ -955,10 +955,8 @@ We have the following scripts for building, testing and coverage:
 - `npm run coverage-html-report` Generates a html report from the coverage data and opens it in a browser on the local machine.
 - `npm run generate-constants` Generates `constants.js` and `events.js` based on the `deltachat-core-rust/deltachat-ffi/deltachat.h` header file.
 - `npm install` After dependencies are installed, runs `node-gyp-build` to see if the native code needs to be rebuilt.
-- `npm run rebuild-all` Rebuilds all the code.
-- `npm run rebuild-core` Builds the `deltachat-core-rust` submodule.
-- `npm run rebuild-bindings` Builds `deltachat.node`.
-- `npm run postinstall` Copies `deltachat.dll` from `deltachat-core-rust` to `build/Release`. Windows only.
+- `npm run node-gyp-build` Tries to load prebuilts and falls back to rebuilding the code.
+- `npm run prebuild` Builds prebuilt binary to `prebuilds/$PLATFORM-$ARCH`. Copies `deltachat.dll` from `deltachat-core-rust` for windows.
```

Comment:
I never understood the prebuild command. Why does it rebuild the core and does prebuild stuff? I prefer having  rebuild-* commands and a prebuildify command to make a prebuild out of the current build. I sometimes only want to rebuild the bindings or the core or do the prebuilds.

Reply:
I can put back some commands to enable you to rebuild each part, but I'm keeping `prebuildify` and `node-gyp-build`.